plugins/builds/steps/get.js

```diff
@@ -21,18 +21,10 @@ module.exports = () => ({
             }
         },
         handler: (request, reply) => {
-            const buildFactory = request.server.app.buildFactory;
-
-            buildFactory.get(request.params.id)
-                .then((buildModel) => {
-                    if (!buildModel) {
-                        throw boom.notFound('Build does not exist');
-                    }
-
-                    const stepModel = buildModel.steps.filter(step => (
-                        step.name === request.params.name
-                    )).pop();
+            const stepFactory = request.server.app.stepFactory;
 
+            stepFactory.get({ buildId: request.params.id, name: request.params.name })
```

Comment:
```suggestion
            return stepFactory.get({ buildId: request.params.id, name: request.params.name })
```

should we add `return`?